source/index.html.md

```diff
@@ -179,6 +179,32 @@ This endpoint finds  customer record by its id
 | 404  | Invalid businessId supplied               |
 
 
+## Upload a csv file containing customer records
+
+This endpoint loads customers by csv files
+
+### HTTP Request
+
+`POST` /enterprise/v1/customers/{businessId}/upload
+
+### Path Parameters
+
+| Parameter            | Description                             |
+|----------------------|-----------------------------------------|
+| `businessId`         | The unique account key of the business  |
+| *string*, *path*     |                                         |
+
```

Comment:
There is a "form" parameter as well. Description should be "The the csv file in a mutlipart form submission"

Reply:
Sorry, the name of the parameter is "file", not "form"